CONTRIBUTING.md

```diff
@@ -90,18 +95,17 @@ python3 -m pip install -U --no-cache-dir  \
         --config-settings="--global-option=-I$(brew --prefix graphviz)/include/" \
         --config-settings="--global-option=-L$(brew --prefix graphviz)/lib/" \
         pygraphviz==1.10
-# Proceed to install other dependencies
-pip install -r requirements.txt
+# Proceed to install dependencies
+uv sync
 ```
 
 ### Running unit tests
 
 The project's unit tests are written using [unittest][unittest] and run using
-the [nose2][nose2] Python package. You can run all unit tests via the `nose2`
-command.
+the [nose2][nose2] Python package. You can run all unit tests via the following command:
 
 ```sh
-nose2
+uv run -- nose2
```

Comment:
Is the `--` necessary? I never used it when I was doing development, but I wasn't necessarily following best practices.